test/controllers/image_controller_test.rb

```diff
@@ -562,6 +562,22 @@ def test_reuse_image_for_glossary_term_post
     assert(glossary_term.reload.images.member?(image))
   end
 
+  def test_reuse_image_for_glossary_term_post_without_thumbnail
+    glossary_term = glossary_terms(:convex_glossary_term)
+    image = images(:commercial_inquiry_image)
+    assert_empty(glossary_term.images)
+    assert_nil(glossary_term.thumb_image)
+    params = {
+      id: glossary_term.id.to_s,
+      img_id: image.id.to_s
+    }
+    login("mary")
+    get_with_dump(:reuse_image_for_glossary_term, params)
```

Comment:
```suggestion
    get(:reuse_image_for_glossary_term, params)
```
https://groups.google.com/g/mo-developers/c/Qi8N2TVSp6c
https://www.pivotaltracker.com/story/show/173424826

Reply:
Done.